sklearn/metrics/cluster/_supervised.py

```diff
@@ -113,14 +108,20 @@ def contingency_matrix(labels_true, labels_pred, *, eps=None, sparse=False):
 
         .. versionadded:: 0.18
 
+    dtype : numeric data type, default=np.int.
+        See the notes on ``eps`` below.
```

Comment:
Makes it a bit long, and duplicates the text, meaning if there is a change in the future on the `eps` explanation it would have to be made twice. I would prefer to leave it as is.

Reply:
Can we specify the following instead (if I don't make a mistake interpreting the note below).
```
Output dtype. Ignore if `eps` is not `None`.
```

Reply:
I added (note the extra "d"):
```
Output dtype. Ignored if `eps` is not `None`.
```